skops/card/_model_card.py

```diff
@@ -10,6 +10,8 @@
 from reprlib import Repr
 from typing import Any, Optional, Union
 
+import matplotlib.pyplot as plt
+import pandas as pd
```

Comment:
Here it is assumed that pandas is installed. However, pandas is not a strict dependency for skops. I think we specifically didn't want to add pandas because it is a "fat" dependency.

Reply:
Yes I thought of it as well, I will try to plot without it.